paddle/fluid/framework/ir/CMakeLists.txt

```diff
@@ -175,7 +175,7 @@ if(WITH_MKLDNN)
   pass_library(softplus_activation_mkldnn_fuse_pass inference DIR mkldnn)
   pass_library(shuffle_channel_mkldnn_detect_pass inference DIR mkldnn)
   pass_library(fc_act_mkldnn_fuse_pass inference DIR mkldnn)
-  pass_library(elt_act_mkldnn_fuse_pass inference DIR mkldnn)
+  pass_library(elt_act_onednn_fuse_pass inference DIR mkldnn)
```

Comment:
As we change it already, please change `elt` to either `eltwise` or `elementwise` as it's more common among other passes.